sound/soc/sof/mediatek/adsp_helper.h

```diff
@@ -0,0 +1,49 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+
+/*
+ * Copyright (c) 2021 MediaTek Corporation. All rights reserved.
+ */
+
+#ifndef __ADSP_HELPER_H__
```

Comment:
This is minor but we've had proper namespace for vendor specific SOF header. Like sound/soc/sof/intel/hda.h has

#ifndef __SOF_INTEL_HDA_H
#define __SOF_INTEL_HDA_H

ADSP is very generic and could conflict with some header in the future.